### ISSUE TITLE
Modding doesn’t do the changes

### ISSUE THREAD

**Issue by vapeclient** [powenn/ModMyIPA#4]
Every time I change bundle ID and/or name and click mod, it doesn’t do the changes

**Comment by powenn** [comment]
Details ?

**Comment by RimuruChan** [comment]
+1

**Comment by RimuruChan** [comment]
Seems that some apps contains a localized infoplist

**Comment by chenrry666** [comment]
> Seems that some apps contains a localized infoplist

yeah! Recently I tried to edit an app's display name and I found it doesn't work. And later I found `en.lproj` folder and edit the `InfoPlist.strings` file in it. Then It finally works.

**Comment by Edouard127** [comment]
Bump same issue, tried multiple times it doesn't change anything
